syncs/github-discussions/schema.sql

```diff
@@ -0,0 +1,73 @@
+CREATE TABLE IF NOT EXISTS github_discussions (
+    repo_id uuid REFERENCES repos(id) ON DELETE CASCADE ON UPDATE RESTRICT,
+    id text,
+    active_lock_reason text,
+    is_answered boolean,
+    answer_id text,
+    answer_chosen_at timestamp with time zone,
+    answer_chosen_by text,
+    author_login text,
+    author_association text,
+    body text,
+    category text,
```

Comment:
Since we have a separate `github_discussion_categories` table, let's save the ID here instead:

```suggestion
    category_id text,
```